database/charts/templates/backups.yaml

```diff
@@ -0,0 +1,86 @@
+---
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: "{{ .Values.repo }}-{{ .Values.zone }}-{{ .Values.backups.component }}"
+spec:
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: "{{ .Values.resources.pvc.size }}"
+  storageClassName: netapp-file-standard
+---
+kind: CronJob
+apiVersion: batch/v1
+metadata:
+  name: "{{ .Values.repo }}-{{ .Values.zone }}-{{ .Values.backups.component }}"
+spec:
+  schedule: "05 8 * * *" # Run daily at 8:xx AM UTC
```

Comment:
lets add an entry in values.yml for schedule, so teams can modify it with values.yml easily

Reply:
Good idea.  Done!